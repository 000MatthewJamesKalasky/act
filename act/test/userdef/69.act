template<pint A, B = 8>
defproc foo (bool x[A], y[B]) { x[0] = y[0]; }

template<pint C, D = 2>
defproc bar <: foo (bool z[C], aa[D]) { }


bar<1> b;
