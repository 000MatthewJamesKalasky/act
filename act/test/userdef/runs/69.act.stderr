In expanding bool
In expanding foo (69.act:1)
In expanding bar<1> (69.act:4)
In expanding ::<Global>
Error on or near line number 8.
 id: A
FATAL: Uninitialized identifier
